src/qml/controls/Wizard.qml

```diff
@@ -11,18 +11,6 @@ Page {
     property var views
     property alias finished: swipeView.finished
     background: null
-    header: RowLayout {
```

Comment:
in 79e959c12f3c7ed3e90619f4cf969325f2c46a0b

a better title is `qml: remove page indicator and navbar from Wizard Control`